src/datatypes/dtype.rs

```diff
@@ -416,15 +419,11 @@ impl Display for DataType {
             DataType::Embedding(inner, size) => {
                 write!(f, "Embedding[{}; {}]", inner.dtype, size)
             }
-            DataType::Image(dtype, mode) => {
-                write!(f, "Image[{}; {:?}]", dtype, mode)
+            DataType::Image(mode) => {
+                write!(f, "Image[{:?}]", mode)
```

Comment:
should be able to use regular display rather than debug display

Reply:
This is an `Option<ImageMode>`, which doesn't implement `Display` but does implement `Debug` with e.g. `Some(RGB)` or `None` displayed.

I can change this to show the `ImageMode` variant if it exists, and maybe `MIXED` otherwise.